config/locales/en.yml

```diff
@@ -1007,10 +1015,14 @@ en:
       who_its_with: With
       reason_for_lateness: Reason for lateness
       reason_for_lateness_note: Note for reason for lateness
+      original_external_deadline: Original external deadline
+      original_internal_deadline: Original internal deadline
     case/ico:
       close: "Record ICO's decision"
       respond: Mark as sent to ICO
       request: "ICO's request"
+    case/ico_foi:
+      record_further_action: Start further ICO request
```

Comment:
Not sure maybe "Start another ICO request"? depends what's being implied. Or "A further"

Reply:
I just followed the design, it is not another ICO request, it is more like more actions are required from ICO rather than new case,